_vendor/src/github.com/gorilla/context/doc.go

```diff
@@ -5,6 +5,12 @@
 /*
 Package context stores values shared during a request lifetime.
 
+Note: gorilla/context, having been born well before `context.Context` existed,
```

Comment:
seem that you change the commit version explicitly? glide couldn't update it automatically.